Source/mpq/mpq_writer.hpp

```diff
@@ -14,14 +14,10 @@
 namespace devilution {
 class MpqWriter {
 public:
-	bool Open(const char *path);
-
-	bool Close(bool clearTables = true);
-
-	~MpqWriter()
-	{
-		Close();
-	}
+	MpqWriter(const char *path);
+	MpqWriter(MpqWriter &&other) = default;
+	MpqWriter &operator=(MpqWriter &&other) = default;

```

Comment:
Now that you've got `unique_ptr` members, which are move-only, the class itself becomes move-only by default. These two lines become superfluous.